website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -2441,5 +2466,8 @@ var Range = ace.require('ace/range').Range;
         chunk.skipLines(2, 1, true);
     }
 
-
 })();
+$(function(){
+  $("wmd-help-button").attr("data-toggle","modal");

```

Comment:
No it can't be, it is selecting by id not class which is why I have it that way.